hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotDiffManager.java

```diff
@@ -170,6 +183,37 @@ public SnapshotDiffManager(ManagedRocksDB db,
     // TODO: [SNAPSHOT] Load jobs only if it is leader node.
     //  It could a event-triggered form OM when node is leader and up.
     this.loadJobsOnStartUp();
+    isNativeRocksToolsLoaded = NativeLibraryLoader.getInstance()
+            .loadLibrary(NativeConstants.ROCKS_TOOLS_NATIVE_LIBRARY_NAME);
+    if (isNativeRocksToolsLoaded) {
+      try {
```

Comment:
I think it would be better to move this try-catch to `initSSTDumpTool`.

Reply:
done